### PR TITLE
116 investigate if there is a way to force tavily searh api to search in GitHub and stackoverflow domain first

### LINKED ISSUE #116: Investigate if there is a way to force tavily searh API to search in github and stackoverflow domain first.

### PR DESCRIPTION
**Pull request by Kranium2002**
Closes #116 

The agent now gives priority to stackoverflow to find solutions, then github and then any other website.

### CODE REVIEW

File: solutionAgent/webcrawler.py [hunk 1]
```diff
@@ -17,9 +21,15 @@ def search(self, query):
...
             
         except Exception as e:
             print(e)
             return None
-        
\ No newline at end of file
+
+if __name__ == '__main__':
+    crawl = WebCrawler()
+    print(crawl.search("""AttributeError: module 'numpy' has no attribute 'object'.
```
Comment (SzymonSt) [inline solutionAgent/webcrawler.py:32]: What is this hardcoded search for?

File: solutionAgent/agent.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def master_agent(self,summary):
         Returns: solution to the logs"""
 
         answer =  self.agent_executor.invoke({"input":f"""You are a software developer who has to provide short summary of available solutions to issues of a software.
-                                     Use websearch tool available to make your answer. You can ask multiple questions from the webagent. Use websearch agent to search about information. You can use the summary provided. Also provide the sources of your solutions.
+                                     Use websearch tool available to make your answer. You can ask multiple questions from the webagent. Use "websearch" agent to search about information use Action: websearch. You can use the summary provided. Also provide the sources of your solutions.
```
Comment (SzymonSt) [inline solutionAgent/agent.py:85]: Do you have any output generated after this change

Comment (Kranium2002) [reply solutionAgent/agent.py:85]: I observed that sometimes there was an error during testing. So, I added this. I do not have any saved output but it worked similar as before.

File: solutionAgent/webcrawler.py (hunk 1 above)
Comment (Kranium2002) [reply solutionAgent/webcrawler.py]: Ahh, sorry I forgot to remove my testing code. Just removed it now.

**Review by SzymonSt — APPROVED** [review]

**Pull request merged**